lib/OnyxUtils.ts

```diff
@@ -381,6 +411,24 @@ function addToEvictionBlockList(key: OnyxKey, connectionID: number): void {
 
     evictionBlocklist[key].push(connectionID);
 }
+/**
+ * It extracts the pure, non-numeric part of a given key.
+ *
+ * For example:
+ * - `getPureKey("report_123")` would return "report_"
+ * - `getPureKey("report")` would return "report"
+ * - `getPureKey("report_")` would return "report_"
+ * - `getPureKey(null)` would return ""
+ *
+ * @param {OnyxKey} key - The key to process.
+ * @return {string} The pure key without any numeric
+ */
```

Comment:
@chrispader what needs to be updated? can you help me I don't recall

Reply:
Just the name and maybe we can change the description to

`It extracts the non-numeric collection identifier of a given key.` or sth.

`getCollectionKey` instead of `getPureKey`